### DISCUSSION TITLE
Email to multiple receivers

### DISCUSSION THREAD

**Discussion by bridei** [Budibase/budibase#4376] [Help]
Hello. Is there any way to comfigure the mail automation to send mail to multiple receivers?
Thanks

**Comment by mjashanks** [comment]
No - but i think we should add this, shouldn't be difficult. I've logged this as a feature request: https://github.com/Budibase/budibase/issues/4385

The only way that you can do it right now is by setting up a REST API datasource to an email provider like Postmark or SendGrid - then you can put whatever you like into the request body
